### PR TITLE
Add CRUX and Micro to README.md

### PR DESCRIPTION
**Pull request by onodera-punpun**

### CODE REVIEW

**Comment by mvdan** [comment]
The new github activity dashboard wasn't showing this and I found it by accident >:( I hate changes.

This is great, but it won't actually do anything unless you use the links somewhere. See the `Packages are available` and `Related projects` lines. You could write something like `Text editor that supports shfmt`.

File: README.md [hunk 1]
```diff
@@ -99,9 +99,11 @@ the parser and the printer. To get started, run:
 [homebrew]: https://github.com/Homebrew/homebrew-core/blob/HEAD/Formula/shfmt.rb
 [nixos]: https://github.com/NixOS/nixpkgs/blob/HEAD/pkgs/tools/text/shfmt/default.nix
 [void]: https://github.com/voidlinux/void-packages/blob/HEAD/srcpkgs/shfmt/template
+[crux]: https://github.com/6c37/crux-ports-git/tree/3.3/shfmt
```
Comment (mvdan) [inline README.md]: also, won't this link get stale due to the `3.3`? is there no `master` equivalent?

Comment (onodera-punpun) [reply README.md]: no sorry there isn't, I'll rename `3.3` to master next CRUX release.

File: README.md [hunk 2]
```diff
@@ -86,22 +86,24 @@ the parser and the printer. To get started, run:
...
+[dockerised-shfmt]: https://hub.docker.com/r/jamesmstone/shfmt/
 [examples]: https://godoc.org/mvdan.cc/sh/syntax#pkg-examples
-[arch]: https://aur.archlinux.org/packages/shfmt/
+[format-shell]: https://atom.io/packages/format-shell
+[go-fuzz]: https://github.com/dvyukov/go-fuzz
 [homebrew]: https://github.com/Homebrew/homebrew-core/blob/HEAD/Formula/shfmt.rb
+[micro]: https://micro-editor.github.io/
+[mksh]: https://www.mirbsd.org/mksh.htm
 [nixos]: https://github.com/NixOS/nixpkgs/blob/HEAD/pkgs/tools/text/shfmt/default.nix
-[void]: https://github.com/voidlinux/void-packages/blob/HEAD/srcpkgs/shfmt/template
```
Comment (mvdan) [inline README.md]: you seem to have deleted this one.

File: README.md [hunk 3]
```diff
@@ -86,22 +86,24 @@ the parser and the printer. To get started, run:
...
+* [micro] - Editor with build-in plugin for `shfmt`
 * [shell-format] - VS Code plugin for `shfmt`
-* [dockerised-shfmt] - A docker image of `shfmt`
 * [vim-shfmt] - Vim plugin for `shfmt`
 
-[posix shell]: http://pubs.opengroup.org/onlinepubs/9699919799/utilities/V3_chap02.html
+[arch]: https://aur.archlinux.org/packages/shfmt/
 [bash]: https://www.gnu.org/software/bash/
-[mksh]: https://www.mirbsd.org/mksh.htm
+[crux]: https://github.com/6c37/crux-ports-git/tree/3.3/shfmt
```
Comment (mvdan) [inline README.md]: this one is still unused - see where `arch` is used

**Comment by mvdan** [comment]
Thanks!

**Pull request merged**